pom.xml

```diff
@@ -1431,7 +1463,7 @@
                     <doctitle>Jersey ${jersey.version} API Documentation</doctitle>
                     <windowtitle>Jersey ${jersey.version} API</windowtitle>
                     <bottom>
-                        <![CDATA[Copyright &#169; 2007-2017,
+                        <![CDATA[Copyright &#169; 2007-2022,
```

Comment:
as soon as you are already modifying this, could you please set 2023 everywhere?